src/models/index.ts

```diff
@@ -28,5 +28,6 @@ export * from "./user-region-info.model";
 export * from "./user-thin-trophy.model";
 export * from "./user-titles-response.model";
 export * from "./user-trophies-earned-for-title-response.model";
+export * from './user-trophies-for-specific-title-response.model';
 export * from "./user-trophy-group-earnings-for-title-response.model";
-export * from "./user-trophy-profile-summary-response.model";
+export * from "./user-trophy-profile-summary-response.model";
```

Comment:
nit: It looks like there's some issue going on with Prettier.

For example, these quotes use single quotes, whereas the other exports are using double quotes. We're also missing the newline at the end of the file.

I'm not sure why CI isn't flagging this ... will investigate.

Reply:
Looks like CI was missing a `format:check` step altogether 😵‍💫 

Added in https://github.com/achievements-app/psn-api/pull/204.

Reply:
Ran `pnpm format` manually on the branch.